src/Numerical_aux/typesdef.f90

```diff
@@ -1084,6 +1084,9 @@ MODULE TypesDef
      LOGICAL                      ,POINTER  :: OutputMask(:)                    !< Mask for variable output
                                                                                 !< .TRUE.  = do output for this variable
                                                                                 !< .FALSE. = do no output for this variable
+     LOGICAL                      ,POINTER  :: PlasticityMask(:)                !< Mask for variable output
```

Comment:
Why is that a pointer when the size is known to be 7?

Reply:
good point.